src/placeos-frontend-loader/api/remote/remote.cr

```diff
@@ -61,9 +78,9 @@ module PlaceOS::FrontendLoader
       File.delete(temp_tar_name)
     end
 
-    def get_hash(hash : String, repository_uri : String, tag : String, branch : String)
+    def get_hash(hash : String, repository_uri : String, tag, branch : String)
       if hash == "HEAD" || hash.nil?
-        if tag != "latest"
+        if (!tag.nil? && tag != "latest")
           hash = get_hash_by_tag(repository_uri, tag)
         elsif branch != "master"
```

Comment:
A bitta danger with none `master` branches here

Reply:
Is there a way to do this without worrying about whether master is or isn't the default?

Reply:
Could just pass it in any way, regardless of `"master"` or not?